### PR TITLE
Fix `useGetMeQuery` usage

### PR DESCRIPTION
**Pull request by mnholtz**
## What does this PR do?

- Closes https://github.com/pixiebrix/pixiebrix-app/issues/3077
- Mis-use of the `useGetMeQuery` endpoint & `useGetQueryState` hook on the `getMe` endpoint resulted in missing `me` information in Automation Anywhere onboarding views (which made it seem like the extension was not being linked successfully, even though it was)

## Reviewer Tips

- `useQueryState` was always `uninitialized` before this bugfix, which resulted in empty `me` data
- See this discussion here: https://pixiebrix.slack.com/archives/C023KL47XV4/p1677610197731169 for why changing the parameter from `null` -> `undefined` fixes this issue

## Discussion

- It's not clear to me why we should be using `useQueryState` at all if we are correctly using `useQuery` for all endpoints with respect to cache behavior. `useQueryState` _composes_ `useQuery` anyways, along with `useQuerySubscription` (`useQueryState` does **not** add any subscriptions, unlike `useQuery` which could result in missing data/unintended bugs if all subscribers are removed)
- I think we should reconsider our usage of `useQueryState` throughout our product

## Future Work

- https://github.com/pixiebrix/pixiebrix-extension/issues/5273
- https://github.com/pixiebrix/pixiebrix-extension/issues/5274

## Checklist

- [ ] Add tests
- [x] Designate a primary reviewer @BLoe

### CODE REVIEW

File: src/options/pages/onboarding/partner/PartnerSetupCard.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ function useInstallUrl() {
 
   const [installURL, isPending] = useAsyncState(async () => {
     const baseUrl = await getBaseURL();
-    const startUrl = new URL(`${baseUrl}start`);
+    const startUrl = new URL(`${baseUrl}/start`);
```
Comment (mnholtz) [inline src/options/pages/onboarding/partner/PartnerSetupCard.tsx:48]: `getBaseUrl` removes all trailing slashes. Not sure why this wasn't caught before.

Comment (BLoe) [reply src/options/pages/onboarding/partner/PartnerSetupCard.tsx:48]: Can you leave a code comment here with this info also? Or maybe add it to the jsDoc comment on the `getBaseUrl()` function or something like that?

**Review by BLoe — APPROVED** [review]

File: src/options/pages/onboarding/partner/PartnerSetupCard.tsx (hunk 1 above)
Comment (twschiller) [reply src/options/pages/onboarding/partner/PartnerSetupCard.tsx:48]: Related PR: https://github.com/pixiebrix/pixiebrix-extension/pull/5259, We shouldn't be manually assembling URLs

Comment (mnholtz) [reply src/options/pages/onboarding/partner/PartnerSetupCard.tsx:48]: Great point, I'll construct it with the URL syntax

**Pull request merged**